src/app/app.module.ts

```diff
@@ -16,6 +16,8 @@ import { VerificationPage } from '../pages/verification/verification';
 import { SearchPipe } from '../pipes/search';
 import { PictureService } from '../services/picture';
 import { MyApp } from './app.component';
+import { AgmCoreModule } from 'angular2-google-maps/core';
```

Comment:
I am not using Sublime, I am using WebStorm.
You can't assume that we are going to use the same IDE - so let's add tslint to make those thing are sorted